### ISSUE TITLE
Specify text colour

### ISSUE THREAD

**Issue by Bilge** [sweetalert2/sweetalert2#2345]
## New feature motivation

There is an option to specify the background colour (`background`) but there appears to be no way to set the foreground (text) colour. WCAG generally requires that background and foreground colours are specified in pairs to ensure correct contrast. 

## New feature description

Add new `colour` option.

## New feature implementation
In addition to colour literals (e.g. `#fff`) it should be valid to specify any other valid CSS property values for `color`, such as `inherit` to use the colour inherited from the stylesheet.
